app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :container_deployment_node
```

Comment:
@durandom ContainerDeploymentNode has a 1 to 1 rel to ContainerDeployment hence vm will also have the same relation with it, or am I wrong?